### PR TITLE
Valentin chion

### PR DESCRIPTION
**Pull request by ValentinChion**
Add bulma to the branch and an example of implementation with instructions & instructionsDisplayer

### CODE REVIEW

File: frontend/src/components/instructions/Instructions.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        }
+    }
+
+//trigger the boolean in state with str as input
+//reusable function
+    triggerDropdown = (str) => () => {
+        this.setState({[str]: !this.state[str]})
+    };
+
+//Put input value in state with name of the input as name of the variable
```
Comment (bdebever) [inline frontend/src/components/instructions/Instructions.js]: /**
* Put input value in state with name of the input as name of the variable
* @param {String} str 
* @returns void 
*/

**Review by bdebever — APPROVED** [review]
Good job dude! Thanks for refactoring the comments ;)

File: backend/package.json [hunk 2]
```diff
@@ -30,11 +26,7 @@
     "cors": "^2.8.5",
     "dotenv-cli": "^1.4.0",
     "express": "^4.16.4",
-<<<<<<< HEAD
-    "mongodb": "^3.1.13",
-=======
     "mongoose": "^5.4.6",
->>>>>>> d487ef0c033deae88032d5da9043627dc3d217a0
```
Comment (NansD) [inline backend/package.json:37]: Lol

File: frontend/src/components/instructions/InstructionsDisplayer.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            <>
+                <div className="box notification is-info">
+                    <div className="content">
+                        <p className="title">Indiquez les détails du sujet</p>
+                    </div>
+                </div>
+                <div className="box">
+                    <div className="columns">
+                        <div className="column is-two-thirds">
+                            <span className="title is-5">Module de l'épreuve:</span>
```
Comment (NansD) [inline frontend/src/components/instructions/InstructionsDisplayer.js:23]: git : triggered

**Comment by NansD** [comment]
image

**Review by NansD — APPROVED** [review]

**Pull request merged**